### PR TITLE
feat(zig): add neotest support for zig

### PR DESCRIPTION
**Pull request by Libsod**
## 📑 Description

Added neotest support for Zig

## ℹ Additional Information

Do we need zig-tools.nvim ? Last activity was 2 years ago, so maybe we can safely remove it ?

### CODE REVIEW

File: lua/astrocommunity/pack/zig/README.md [hunk 1]
```diff
@@ -6,9 +6,11 @@
 - curl / wget (required by zig-tools)
 - Neovim (>= 0.7.x)
 - ripgrep (>= 11.0)
+- Zig (>= 0.12.0)
```
Comment (mehalter) [inline lua/astrocommunity/pack/zig/README.md:9]:
```suggestion
- Zig (>= 0.12.0 required for Neotest support)
```

Specify that the version is only required for the Neotest integration

**Review by mehalter — CHANGES_REQUESTED** [review]
Thanks! Small nitpick about the README

File: lua/astrocommunity/pack/zig/README.md (hunk 1 above)
Comment (Libsod) [reply lua/astrocommunity/pack/zig/README.md:9]: Oh, sorry, thank you for suggestion

**Review by mehalter — APPROVED** [review]
LGTM!

**Pull request merged**